manifests/image-references

```diff
@@ -14,3 +14,7 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-cluster-policy-controller:v4.3
+  - name: tools
```

Comment:
keep a storage team indication and name here.

Reply:
done